pkg/packagekit/autoupdates.jsx

```diff
@@ -79,15 +81,15 @@ class DnfImpl extends ImplBase {
             // - by default this runs every day (OnUnitInactiveSec=1d), but the timer can be changed with a timer unit
             //   drop-in, so get the last line
             cockpit.script("set -e; if rpm -q " + this.packageName + " >/dev/null; then echo installed; fi; " +
-                           "if grep -q '^[ \\t]*upgrade_type[ \\t]*=[ \\t]*security' /etc/dnf/automatic.conf; then echo security; fi; " +
-                           "TIMER=dnf-automatic-install.timer; " +
-                           "if systemctl --quiet is-enabled dnf-automatic-install.timer 2>/dev/null; then echo enabled; " +
-                           "elif systemctl --quiet is-enabled dnf-automatic.timer 2>/dev/null && grep -q '^[ \t]*apply_updates[ \t]*=[ \t]*yes' " +
-                           "    /etc/dnf/automatic.conf; then echo enabled; TIMER=dnf-automatic.timer; " +
-                           "fi; " +
-                           'OUT=$(systemctl cat $TIMER 2>/dev/null || true); ' +
-                           'echo "$OUT" | grep "^OnUnitInactiveSec= *[^ ]" | tail -n1; ' +
-                           'echo "$OUT" | grep "^OnCalendar= *[^ ]" | tail -n1; ',
+        "if grep -q '^[ \\t]*upgrade_type[ \\t]*=[ \\t]*security' /etc/dnf/automatic.conf; then echo security; fi; " +
+        "TIMER=dnf-automatic-install.timer; " +
+        "if systemctl --quiet is-enabled dnf-automatic-install.timer 2>/dev/null; then echo enabled; " +
+        "elif systemctl --quiet is-enabled dnf-automatic.timer 2>/dev/null && grep -q '^[ \t]*apply_updates[ \t]*=[ \t]*yes' " +
+        "    /etc/dnf/automatic.conf; then echo enabled; TIMER=dnf-automatic.timer; " +
+        "fi; " +
+        'OUT=$(systemctl cat $TIMER 2>/dev/null || true); ' +
+        'echo "$OUT" | grep "^OnUnitInactiveSec= *[^ ]" | tail -n1; ' +
+        'echo "$OUT" | grep "^OnCalendar= *[^ ]" | tail -n1; ',
```

Comment:
I'm not a big fan of outdenting this, this looks just wrong?

Reply:
Unintentional indenting of my editor, sorry!